installing/installing-fips.adoc

```diff
@@ -83,4 +83,5 @@ If you are using Azure File storage, you cannot enable FIPS mode.
 
 To apply `AES CBC` encryption to your etcd data store, follow the xref:../security/encrypting-etcd.adoc#encrypting-etcd[Encrypting etcd data] process after you install your cluster.
 
-If you add {op-system-base} nodes to your cluster, ensure that you enable FIPS mode on the machines before their initial boot. See xref:../machine_management/adding-rhel-compute.adoc#adding-rhel-compute[Adding RHEL compute machines to an {product-title} cluster] and link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/chap-federal_standards_and_regulations#sec-Enabling-FIPS-Mode[Enabling FIPS Mode] in the {op-system-base} 7 documentation.
+If you add {op-system-base} nodes to your cluster, ensure that you enable FIPS mode on the machines before their initial boot. See xref:../machine_management/adding-rhel-compute.adoc#adding-rhel-compute[Adding RHEL compute machines to an {product-title} cluster]
```

Comment:
@knewcomerRH why is this... I know that you and I discussed this in the past and we have valid reasons for this with RHCOS / Install dependencies; but for RHEL 7 & 8 workers why is this note needed; the RHEL 7 / 8 docs don't require this to pass FIPS validation?

Reply:
Hi @sferich888 If the fips is enabled on Rhel before OCP installed, then only fips compliant crypto algorithm/packages will be used/installed.
You could see from https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp4-rhel-scaleup-runner/13656/consoleFull. There is TASK will check the Rhel nodes' fips status, and will fail directly if fips status(of RHCOS and Rhel nodes) doesn't match. It comes from a customer bug. Sorry, I failed to find the bug id.
TASK [openshift_node : Fail if host FIPS status does not match cluster FIPS status] ***

Reply:
As we discussed in chat a while back, the recommendation for booting with FIPS on day 1 is currently required for RHCOS / control plane nodes in order to ensure that all keys and certs are generated with FIPS compliant crypto. As TLS certs are generated on RHEL 7 & RHEL 8 worker nodes for encrypting communication with control plane nodes, I believe it's appropriate to require this for these worker nodes as well.